include/Core/Base/Mixin/Renderer.php

```diff
@@ -432,9 +432,16 @@ protected function render_cron_settings() {
 			</td>
 		</tr>
 
-		<tr>
+		<tr class="<?php if ( ! empty( $pro_class ) ) { echo sanitize_html_class( $pro_class ); } ?>" style="display: none;">
```

Comment:
You can remove the `empty` check and just print it after passing through `sanitize_html_class`.

Reply:
> you can remove the `empty` check and just print it after passing through `sanitize_html_class`.

Just realized that `sanitize_html_class` handles empty string. Removed the explicit check.